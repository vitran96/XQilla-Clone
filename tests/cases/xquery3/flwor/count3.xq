for $s in ("oxford", "london")
for $p in (
<p name="courgettes" store="london"/>,
<p name="tomatoes" store="london"/>,
<p name="aubergine" store="oxford"/>,
<p name="runner beans" store="london"/>,
<p name="dwarf beans" store="oxford"/>,
<p name="potatoes" store="london"/>
)
where $p/@store = $s
count $c
let $column := $c mod 2
order by $column
return <product column="{$column}">{ $p/@name, $p/@store }</product>
