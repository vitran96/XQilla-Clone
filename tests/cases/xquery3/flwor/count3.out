<product column="0" name="dwarf beans" store="oxford"/><product column="0" name="tomatoes" store="london"/><product column="0" name="potatoes" store="london"/><product column="1" name="aubergine" store="oxford"/><product column="1" name="courgettes" store="london"/><product column="1" name="runner beans" store="london"/>